src/Language/Swift.ts

```diff
@@ -528,13 +528,13 @@ class SwiftRenderer extends ConvenienceRenderer {
                     this.emitLine("self = try JSONDecoder().decode(", this.swiftType(c), ".self, from: data)");
                 });
                 this.ensureBlankLine();
-                this.emitBlock(["init?(_ json: String, using encoding: String.Encoding = .utf8) throws"], () => {
-                    this.emitLine("guard let data = json.data(using: encoding) else { return nil }");
+                this.emitBlock(["init(_ json: String, using encoding: String.Encoding = .utf8) throws"], () => {
+                    this.emitLine("guard let data = json.data(using: encoding) else { throw URLError(.cannotParseResponse) }");
```

Comment:
Can `URLError(.cannotParseResponse)` be written as `URLError.cannotParseResponse`? Why is `URLError.badURL` written differently?

Reply:
turns out that same applies for both, except that it didn't complain in playground but it did in a project, weird swift